v1/tools/toolchain/windriver/cmake/FindCURL.cmake

```diff
@@ -0,0 +1,66 @@
+# - Find curl
```

Comment:
If this commit fixes the GitHub issue about finding curl during WindRiver cross-compile, then let's make sure the issue # goes into the commit message so that everything gets linked up properly.

Reply:
I think the current WRL problems are unrelated to this, I'll hold off on linking this to an open issue until we go through existing open issues.

Reply:
Got it.